ols/utils/config.py

```diff
@@ -124,30 +124,3 @@ def load_config_from_env() -> None:
             m.name = model
             oai_provider.models[m.name] = m
         llm_config.providers[oai_provider.name] = oai_provider
-
-
-# def load_config(config_file):
-#     try:
-#         with open(config_file, "r") as f:
-#             data = yaml.safe_load(f)
-#             c=Config(data)
-#     except Exception as e:
-#         print(f"Failed to load config file {config_file}: {str(e)}")
-#         print(traceback.format_exc())
-#         exit(1)
-#     try:
-#         c.validate()
-#     except Exception as e:
-#         print(f"Failed to validate config file {config_file}: {str(e)}")
-#         print(traceback.format_exc())
-#         exit(1)
-#     global config
-#     global default_logger
-#     global feedback_logger
-#     global conversation_cache
-#     config = c
-#     default_logger = Logger(logger_name="default", logfile=c.ols_config.logger_config.default_filename,log_level=c.ols_config.logger_config.default_level).logger
-#     feedback_logger = Logger(logger_name="feedback", logfile=c.ols_config.logger_config.feedback_filename,log_level=c.ols_config.logger_config.feedback_level).logger
-
-#     conversation_cache = CacheFactory.conversation_cache(c.ols_config.conversation_cache)
-#     return c
```

Comment:
this is also related to the future implementation of the yaml config loading.  @syedriko may want to start fresh, but i think it's better if we let him remove this as part of his work to introduce the config.yaml loading officially.

Reply:
Is necessary to have it in code though? It is more like a snippet that should be part of the Jira story.

Reply:
it's not, but it's there now and it'll be removed shortly (this sprint) by the work Sergey is doing, so rather than remove it(and make him dig it up from git history for reference) or moving it elsewhere, let's just let him remove it as part of the work he's already doing, when he's ready.

Reply:
(The reason it's here is this is code i wrote when testing the config work i was doing, and knowing that it would become relevant again when we were ready to adopt my config work, i left it in place but commented out for the future time when we'd want it again, or at least want to use it as a reference.  I'm not recommending it as a standard practice)

Reply:
I still think a snippet is a better option here, leaving Sergey already a clean slate for work. But ok. I'll close this and open a new one with the test one (as its oneliner, I'm too lazy to revert and force push here:)).